src/transaction.js

```diff
@@ -41,6 +41,11 @@ function Transaction (network = networks.bitcoin) {
     this.overwintered = 0
     this.versionGroupId = 0  // 0x03C48270 (63210096) for overwinter
     this.expiryHeight = 0
+    // ZCash version >= 4
+    this.valueBalance = 0
```

Comment:
please document each of these fields, so a reader does not need to dive into the 100+ page zcash spec in order to understand at a high level

Reply:
Most of these are for shielded transactions. I'm adding comments for those that are interesting for transparent txs